### PR TITLE
[Component] [Formbuilder] + fix integration of elements

### PR DESCRIPTION
**Pull request by fkroockmann**
- fix integration of all elements

### CODE REVIEW

File: src/tb/component/formbuilder/form/element/templates/password.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<label for="">{{ element.getLabel() }}</label>
```
Comment (mickaelandrieu) [inline src/tb/component/formbuilder/form/element/templates/password.twig]: hmm, maybe it's time add a real ``for`` here ?

Comment (ndufreche) [inline src/tb/component/formbuilder/form/element/templates/password.twig]: Yes, I'm agree with @mickaelandrieu

**Comment by fkroockmann** [comment]
thanks for review, you can merge :)

**Comment by ndufreche** [comment]
@fkroockmann you need to add for attribute into label tag
```twig
<label for="{{ element.getKey() }}" class="label">{{ element.getLabel() }}</label>
```

**Comment by fkroockmann** [comment]
@ndufreche i fix it, you can merge :)

**Comment by ndufreche** [comment]
:+1:  thanks

**Pull request merged**